### PR TITLE
make one job run more models

### PR DESCRIPTION
**Pull request by cwen0**
@zhouqiang-cl PTAL

### CODE REVIEW

File: oasis/job.py [hunk 1]
```diff
@@ -2,31 +2,71 @@
...
+            self.running_models.append(model_runner)
+            self.threads[md.get("name")] = t
+
+    def close(self):
+        logger.info("[job-id:{job_id}] start to stop"
+                    .format(job_id=self.id))
+        for model in self.running_models:
+            model.close()
+
+        for key in self.threads:
```
Comment (zhouqiang-cl) [inline oasis/job.py]: key-> module?

File: oasis/job.py [hunk 2]
```diff
@@ -2,31 +2,71 @@
...
+        self.models = models
         self.slack_channel = slack_channel
-        self.timeout = timeout
-        self.config = config
-    # def __init__(self, data):
-    # self.__dict__ = json.loads(data)
+        self.running_models = []
+        self.threads = dict()
 
     def __iter__(self):
```
Comment (zhouqiang-cl) [inline oasis/job.py]: when to use `__iter__`

File: oasis/job.py (hunk 1 above)
Comment (cwen0) [reply oasis/job.py]: key is the name of the model

Comment (zhouqiang-cl) [reply oasis/job.py]: so, name it as module?

Comment (cwen0) [reply oasis/job.py]: every model will run in one single thread, so, I add every model threads to the job for this job can close all model threads.  I don't understand what you mean.

**Comment by cwen0** [comment]
@zhouqiang-cl PTAL again

**Pull request merged**